.github/workflows/wildfly-it-tests-manual-bootable.yml

```diff
@@ -0,0 +1,84 @@
+        type: string
+
+jobs:
+  wildfly-it-tests-manual-bootable:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+        java-version: ['17', '21']
+        java-distribution: ['adopt']
+    steps:
+      - name: Checkout eap-microprofile-test-suite
+        uses: actions/checkout@v4
+      - name: Set up JDK ${{ matrix.java-distribution }} ${{ matrix.java-version }}
+        uses: actions/setup-java@v4
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: ${{ matrix.java-distribution }}
+          cache: 'maven'
+      - name: Checkout and build WildFly
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ inputs.wildFly-repo }}
+          ref: ${{ inputs.wildFly-ref }}
+          path: 'local-wildfly'
+        if: ${{ inputs.build-wildFly == true }}
+      - name: Build WildFly
+        run: cd local-wildfly ; mvn clean install -DskipTests --batch-mode; cd ..
+        if: ${{ inputs.build-wildFly == true }}
+      - name: Set the value for the ENV VAR that stores the WildFly version
+        run: |
+          echo "MAVEN_ARGS_WILDFLY_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout -f local-wildfly/pom.xml)" >> "$GITHUB_ENV"
+        if: ${{ inputs.build-wildFly == true }}
+      - name: Set the value for the ENV VAR that stores the WildFly snapshot distribution version property
+        run: |
+          echo "MAVEN_ARGS_WILDFLY_DIST_VERSION=-Dversion.org.jboss.wildfly.dist=${{ env.MAVEN_ARGS_WILDFLY_VERSION }}" >> "$GITHUB_ENV"
+        if: ${{ inputs.build-wildFly == true }}
+      - name: Set the value for the ENV VAR that stores the WildFly snapshot distribution location property
+        run: |
+          echo "MAVEN_ARGS_WILDFLY_DIST_LOCATION=-Djboss.dist=${PWD}/local-wildfly/dist/target/wildfly-${{ env.MAVEN_ARGS_WILDFLY_VERSION }}" >> "$GITHUB_ENV"
+        if: ${{ inputs.build-wildFly == true }}
+      - name: Set the value for the WildFly JAR (Bootable JAR) Maven plugin version
+        run: |
+          echo "MAVEN_ARGS_WILDFLY_JAR_MAVEN_PLUGIN_VERSION=-Dversion.org.wildfly.jar.plugin=${{ inputs.wildfly-jar-maven-plugin-version }}" >> "$GITHUB_ENV"
+        if: ${{ ! inputs.wildfly-jar-maven-plugin-version == '' }}
+      - name: Build and run integration tests (${{ matrix.java-distribution }} ${{ matrix.java-version }}) against latest WildFly, on ${{ matrix.os }}
+        if: ${{ matrix.os  == 'ubuntu-latest' }}
+        run: mvn clean verify --batch-mode -fae ${{ env.MAVEN_ARGS_WILDFLY_DIST_LOCATION }} ${{ env.MAVEN_ARGS_WILDFLY_DIST_VERSION }} ${{ env.MAVEN_ARGS_WILDFLY_JAR_MAVEN_PLUGIN_VERSION }}
+      - name: Build and run integration tests (${{ matrix.java-distribution }} ${{ matrix.java-version }}) against latest WildFly, on ${{ matrix.os }} (excluding Docker)
+        if: ${{ matrix.os  == 'windows-latest' }}
```

Comment:
What is the plan for Windows? My understanding is that windows is kept here, but is unsupported?

Reply:
Thanks @marekkopecky - yes, I'd leave Windows out ATM, as per the PR description, in order to have something to run upstream CI checks manually _soon_.
Then we can expand to Windows with a separate task. WDYT?